crates/env/src/engine/on_chain/pallet_revive.rs

```diff
@@ -937,6 +937,22 @@ impl TypedEnvBackend for EnvInstance {
         ext::gas_limit()
     }
 
+    fn gas_price(&mut self) -> u64 {
+        ext::gas_price()
+    }
+
+    fn ref_time_left(&mut self) -> u64 {
+        ext::ref_time_left()
```

Comment:
This was renamed to `gas_left` in `uapi` at https://github.com/paritytech/polkadot-sdk/pull/9968, A TODO comment for updating this in the future may be useful (@cmichi not sure whether or not we're close to syncing past that commit/PR).